src/dodal/devices/slits.py

```diff
@@ -15,3 +15,17 @@ def __init__(self, prefix: str, name: str = "") -> None:
             self.x_centre = Motor(prefix + "X:CENTRE")
             self.y_centre = Motor(prefix + "Y:CENTRE")
         super().__init__(name)
+
+
+class SpecialSlitsi22(StandardReadable):
+    """
+    these are unlikely to be moved during a scan
+    """
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        with self.add_children_as_readables():
+            self.x = Motor(prefix + "X:RBV")
```

Comment:
https://github.com/bluesky/ophyd-async/blob/8273a2e0980fc586db38abb15fc6d354e0d40e68/src/ophyd_async/epics/motor.py#L54

This looks wrong, or otherwise incompatible with the Motor class- can you instantiate this on the beamline?
Line 64 of the Motor definition adds an RBV signal, why did you add the RBV suffix here?

Reply:
I forgot it's in the motor, it did not connect correctly. will investigate further